### ISSUE TITLE
Use scraproxy to not be blacklisted

### ISSUE THREAD

**Issue by frankrousseau** [cozy-labs/konnectors#279]
@fabienvauchelles wrote a proxy to allow scrapping without blacklisting. Maybe there is something that konnectors could reuse from it. I didn't make extensive research. But I post the link here for discussion and brainstorm about it.

http://scrapoxy.io/

**Comment by fabienvauchelles** [comment]
Yep :) The proxy starts/stop instances on AWS (or OVH/DigitalOcean) and routes your request throught them.

**Comment by nicofrand** [comment]
Since we send the login/passwords tuple to the website to connect to it, using a proxy would mean giving it our passwords (even through HTTPS), I am not sure I trust this (conceptually, not just for scaproxy which seems nice ;)).
